ai-baseline/openai-baseline/app/static/__vectorStore.py

```
@@ -0,0 +1,238 @@
+    try:
+        vector_store_files = vs_api.files.list(vector_store_id=vector_store_id)
+        
+        for file_item in vector_store_files.data:
+            try:
+                # Get file details to get the filename
+                file_detail = client.files.retrieve(file_item.id)
+                if hasattr(file_detail, 'filename') and file_detail.filename:
+                    existing_filenames.add(file_detail.filename)
+            except Exception as e:
+                # If we can't get filename, skip this file
+                pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)